_posts/2024-02-01-december-january.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Desember 2023, januar 2024
+author: Bestem0r
+---
+
+Trudde du kanskje det var over og ut for utviklarbloggen då me ikkje kom med ein utgåve i desember? 🤔 Frykt ikkje! Etter ein lang juleferie er me nemleg tilbake med ei dobbeltutgåve av utviklarbloggen! 🎉 I denne utgåva kan du lese om alt me har jobba med i desember og januar! 🥳
+
+# Konfetti når ein opprettar ein brukar
+
+For å feire nye abakule, har me no lagt til konfetti når ein opprettar ein brukar! 🎊 Litt ekstra stas altså! 🥳
+
+![Konfetti](/images/posts/2024-02-01-konfetti.png)
+
+# Grupper i sidemenyen
+
+Endeleg kan du sjå alle abakus sine grupper i sidemenyen! 🤩 Dette inkluderer både styrer, grupper og komitear!
+
+| Før                                                                                                           | Etter                                                                                                           |
+|---------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| ![Før](https://user-images.githubusercontent.com/13599770/297894667-1e4d07d0-5d17-40b8-8202-898adeaad0e9.png) | ![Etter](https://user-images.githubusercontent.com/13599770/297894715-7d6ed3c3-157a-44de-b5bc-7aa8bca95072.png) |
```

Comment:
Dette har vel vært her en stund. Det som er nytt er vel at de er synlige selv for de som ikke er logget inn?

Reply:
Ah right. Removing the section then, since it's not really news